view/app-jarvis.js

```diff
@@ -139,7 +139,7 @@ app.controller('MainController', function($scope,$location,$rootScope,$http) {
 					messageObj.message = message;
 					$scope.messageStack.push(messageObj);
 				} else if ((status === 'success' || status) && message === 'Here are your reminders : ') {
-					console.log("In show reminder");
+					console.log('In show reminder');
```

Comment:
please remove all ***console.log*** statements. I know, these were added in the previous PR, but I guess, we don't need them anymore!